test/DGmethods_old/soundings/script_used_to_make_soundings.jl

```diff
@@ -0,0 +1,35 @@
+using DelimitedFiles
```

Comment:
Does this file need to be committed if you've converted the data? If so, please call it something else... `soundings_dat_to_nc.jl` or something?

Reply:
Done